### ISSUE TITLE
Support scala 2.13, drop dependencies which are reluctant to migrate (cats? scalameta? fastparse?) 

### ISSUE THREAD

**Issue by pshirshov** [7mind/izumi#266]
At the moment we are blocked by:

- [x] `scalameta`: https://github.com/scalameta/scalameta/issues/1693 . Another issue is lack of comments support. So #280 
- [x] `cats`/`cats-effect`: https://github.com/typelevel/cats/issues/2267
- [x] `circe`: https://github.com/circe/circe/issues/770
- [x] `http4s`: https://github.com/http4s/http4s/issues/1797
- [x] `fastparse` https://github.com/lihaoyi/fastparse/issues/188 https://github.com/lihaoyi/fastparse/pull/194

2.13 preparations: #265
